lib/screens/v2/candidate_generator/widgets/elevator_closures.ex

```diff
@@ -64,9 +67,11 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ElevatorClosures do
     end
   end
 
-  defp get_icon_map(elevator_closures) do
+  defp get_icon_map(elevator_closures, home_parent_station_id) do
     elevator_closures
     |> get_parent_station_ids_from_entities()
+    |> MapSet.new()
+    |> MapSet.put(home_parent_station_id)
```

Comment:
So this has to do with one part of my solution that might be a little hacky, and takes some explanation.

I needed to sort the list of alerts by whether they inform an elevator at a station on a connecting line with the home station. Since we already figure out the lines serving each stop in order to build the list of icons to show, I figured I could use those.

However we only built the station ID -> icons mapping for stations that have an elevator closure alert. If there were no alerts at the home station, my code would break.

This change makes sure we always include the home station in the station ID -> icons mapping, even if there are no elevator closures at it.

See here: https://github.com/mbta/screens/pull/1265/files#diff-61774d8dd847cfd8616f84ebf250ebd552da56f8b30d39b3b6a7b6aa05127a9eR317-R336

Reply:
Ahh I see. I think long term, I could see Screenplay helping with this. But we'll cross that bridge when we get there.